### PR TITLE
Adapt the new metadata standardization in Istio 1.24

### LINKED ISSUE #12643: [Feature] adapt Istio ALS new peer metadata attributes
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

https://github.com/istio/istio/pull/53274/files

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a pull request to implement this on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #12643 .
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: .github/workflows/skywalking.yaml [hunk 1]
```diff
@@ -555,26 +555,31 @@ jobs:
...
+              ISTIO_VERSION=1.22.0
+              KUBERNETES_VERSION=28
+          - name: Istio Metrics Service 1.23.0
             config: test/e2e-v2/cases/istio/metrics/e2e.yaml
             env: |
-              ISTIO_VERSION=1.18.0
-              KUBERNETES_VERSION=25
+              ISTIO_VERSION=1.23.0
+              KUBERNETES_VERSION=28
+          - name: Istio Metrics Service 1.23.0
```
Comment (wu-sheng) [inline .github/workflows/skywalking.yaml:578]:
```suggestion
          - name: Istio Metrics Service 1.24.0
```

File: test/e2e-v2/cases/profiling/ebpf/access_log/traffic-gen.yaml [hunk 2]
```diff
@@ -33,8 +33,14 @@ spec:
     spec:
       containers:
         - name: trafficgenerator
-          image: elswork/wrk
-          command: ["wrk", "-t1", "-c1", "-d20m", "http://productpage:9080/productpage"]
```
Comment (kezhenxu94) [inline test/e2e-v2/cases/profiling/ebpf/access_log/traffic-gen.yaml:37]: FYI @mrproliu , let’s avoid using such images that was not updated for 6 years and doesn’t work on Apple's M chip machines. Please consider removing this in satellite repo also.

File: .github/workflows/skywalking.yaml [hunk 3]
```diff
@@ -555,26 +555,31 @@ jobs:
...
-              KUBERNETES_VERSION=25
+              ISTIO_VERSION=1.23.0
+              KUBERNETES_VERSION=28
+          - name: Istio Metrics Service 1.24.0
+            config: test/e2e-v2/cases/istio/metrics/e2e.yaml
+            env: |
+              ISTIO_VERSION=1.24.0
+              KUBERNETES_VERSION=28
 
           - name: Rover with Istio Process 1.15.0
```
Comment (kezhenxu94) [inline .github/workflows/skywalking.yaml:584]: @mrproliu consider upgrading this istio version to >= 1.20.0 and test whether the eBPF module needs updated, 1.15 is quite old and I don’t think we would like to maintain it anymore

File: test/e2e-v2/cases/profiling/ebpf/access_log/traffic-gen.yaml (hunk 2 above)
Comment (wu-sheng) [reply test/e2e-v2/cases/profiling/ebpf/access_log/traffic-gen.yaml:37]: Yes, please update them after this gets merged.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**